drop-transfer/src/ws/server/mod.rs

```diff
@@ -568,6 +568,10 @@ impl FileXferTask {
     ) {
         let init_res = match downloader.init(&self).await {
             Ok(init) => init,
+            Err(crate::Error::Canceled) => {
+                warn!(logger, "File cancelled on download init stage");
+                return;
+            }
```

Comment:
So in this case if the file has already been cancelled, the application does not get an indication besides the log anymore?

Reply:
That's a nice catch @Lipt0nas . I think the message is fine however the error mechanism should go back and be improved separately if at all

Reply:
I left a TODO comment, since this might become a non-issue when we implement the auto retries